File: basic-example/1.Comments-In-Different-Lang/Comments.kt
fun main () {
  // run code: kotlinc Comments.kt -include-runtime -d Comments.jar &&  java -jar Comments.jar
  // 最好升级一下你的 kotlin 版本，不然可能会有报错 ”没有主清单文件"
  val x = 5 + 5;
  // 这里也会报错,如果想跑，请注释下一行
  /*abc /*dd*/ */
  println("Is `x` 10 or 100? x = $x");
}

File: basic-example/1.Comments-In-Different-Lang/Comments.rs
fn main() {
    // 单行注释
    // run code: rustc Comments.rs && ./Comments
    let x = 5 + 5;
    // 多行注释，块注释。特性: 可以嵌套，TS和Kotlin都不支持
    /* 这里必须加 ! , 如果不加则会报无法调用 marco 的错误 /* help: use `!` to invoke the macro */ */
    println!("Is `x` 10 or 100? x = {}", x);
    // 模板字符串不同于 JS、Ts 的 const str = `abc ${var}`
    // 也不同于 Kotlin 的 val str = "abc $var"
}
